common/scripted_effects/00_scripted_effects.txt

```diff
@@ -259,13 +259,25 @@ add_trait_one_eyed_effect = {
 		
 	}
 }
-
+#WC
 add_trait_one_handed_effect = {
 	if = {
 		limit = { NOT = { trait = one_handed } }
 		add_trait = one_handed
-		hidden_tooltip = { character_event = { id = RIP.11502 } }
-		add_trait_severely_injured_effect = yes
+		#hidden_tooltip = { character_event = { id = RIP.11502 } }
+		#add_trait_severely_injured_effect = yes
```

Comment:
Nani? Why it's commented?

Reply:
its pdx stuff so i just disabled it

Reply:
This effect is created to get a one-handed trait and show a notification. It's useful and I don't get why you changed it instead of creating new effect you want.

Reply:
it wasnt used and the name fits and it replaces add_trait = one_handed so didnt want to make any changes to that

Reply:
> it wasnt used and the name fits and it replaces add_trait = one_handed so didnt want to make any changes to that

But you did changes. It doesn't show notification and doesn't add injury trait.
So for example, if we're going to work on battle events, you need to revert these changes you made and restore the original effect.

Reply:
why? the effect wasnt used in any events?

Reply:
> why? the effect wasnt used in any events?

I plan to use it.